darts/tests/test_metrics.py

```diff
@@ -89,14 +106,26 @@ def helper_test_multiple_ts_duplication_equality(self, metric, **kwargs):
             self.assertAlmostEqual(metric(s1, s2, **kwargs, reduction=np.mean, inter_reduction=np.max),
                                    metric(s11, s22, **kwargs, reduction=np.mean, inter_reduction=np.max))
 
-    def helper_test_nan(self, metric):
-        # univariate
-        non_nan_metric = metric(self.series1[:9] + 1, self.series2[:9])
-        nan_series1 = self.series1.copy()
-        nan_series1._xa.values[-1,:,:] = np.nan
-        nan_metric = metric(nan_series1 + 1, self.series2)
-        self.assertEqual(non_nan_metric, nan_metric)
-        # multivariate (TODO)
+    def helper_test_nan(self, metric, **kwargs):
+        test_cases, kwargs = self.get_test_cases(**kwargs)
+
+        for s1, s2 in test_cases:
+            # univariate
+            non_nan_metric = metric(s1[:9] + 1, s2[:9])
+            nan_s1 = s1.copy()
+            nan_s1._xa.values[-1, :, :] = np.nan
+            nan_metric = metric(nan_s1 + 1, s2)
+            self.assertEqual(non_nan_metric, nan_metric)
+
+            # multivariate
```

Comment:
```suggestion
            # multivariate + multi-TS
```
Very nice extension, thanks!